manifests/install.pp

```diff
@@ -0,0 +1,27 @@
+# @summary Install pulpcore packages
+# Currently this is done with pip3. This should have the option to also install via RPMs.
+# @api private
+class pulpcore::install {
+
+  $system_packages = ['gcc', 'postgresql-server', 'postgresql-devel', 'python3-pip', 'python3-devel']
```

Comment:
`postgresql-server` should already be done by the postgresql server class in database.pp.